impress-org/givewp
# fix: keep MultiStep goal stats from overflowing parent container

Resolves [GIVE-852]

## Description
We want to ensure the goal stats don't overflow their parent container The elements should flow responsively, but at the smallest screen size necessary.

## Affects
- MultiStep-goal

## Visuals

https://github.com/user-attachments/assets/933aabbc-6860-47ec-a178-57c4c3361d1a


## Testing Instructions
- View a MultiStep form.
- Reduce screen size.
- Ensure stats do not overflow the container.

## Pre-review Checklist



[GIVE-852]: https://stellarwp.atlassian.net/browse/GIVE-852?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
